srcpkgs/flavours/template

```diff
@@ -9,7 +9,7 @@ maintainer="Numair Dubas <Numair2644@protonmail.com>"
 license="MIT"
 homepage="https://github.com/Misterio77/flavours"
 distfiles=${homepage}/archive/refs/tags/v${version}.tar.gz
```

Comment:
```suggestion
distfiles=https://github.com/Misterio77/flavours/archive/refs/tags/v${version}.tar.gz
```

Reply:
Thank you for the suggestions. Should I squash the commits?

Reply:
yes please

Reply:
Done. I also used an email I found on your packages because `xlint` was complaining about a noreply address.